src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs

```diff
@@ -333,6 +333,55 @@ public void Stays_on_full_sync(string configWildcard, bool stickToFullSyncAfterF
             }
         }
 
+        [Test]
+        public void Diagnostics_mode_is_not_enabled_by_default()
+        {
+            foreach (string configFile in Resolve("^spaceneth.cfg"))
+            {
+                ConfigProvider configProvider = GetConfigProviderFromFile(configFile);
+                IInitConfig initConfig = configProvider.GetConfig<IInitConfig>();
+                initConfig.DiagnosticMode.Should().Be(DiagnosticMode.None, configFile);
+            }
+        }
+
+        [Test]
+        public void Migrations_are_not_enabled_by_default()
+        {
+            foreach (string configFile in Resolve("*"))
```

Comment:
take this `"*"` string to `TestCase` like u did later to leave no room for interpretation what it is

Reply:
fixed